### PR TITLE
Update code samples for webdriverio 

### PR DESCRIPTION
**Pull request by nurdtechie98**
* Move to webdriverio v5
* Change Directory structure
* Update Readme

### CODE REVIEW

File: ios/run_parallel_test/specs/single_test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    var textOutput = await $(`~Text Output`);
+    await textOutput.waitForDisplayed({ timeout: 30000 });
+    var value = await textOutput.getText();
+
+    if (value === "hello@browserstack.com")
+      assert(true)
+    else
+      assert(false)
+  });
+});
```
Comment (darshilgada24) [inline ios/run_parallel_test/specs/single_test.js:23]: add a new line

File: ios/run_parallel_test/parallel.conf.js [hunk 2]
```diff
@@ -17,12 +17,12 @@ exports.config = {
   },
 
   capabilities: [{
-    "device": "iPhone 7"
+    "device": "iPhone 11 Pro"
```
Comment (darshilgada24) [inline ios/run_parallel_test/parallel.conf.js:20]: specify os_versions

File: ios/run_multiple_test/multiple.conf.js [hunk 3]
```diff
@@ -4,19 +4,19 @@ exports.config = {
...
-    './tests/specs/multiple_test.js'
+    './run_multiple_test/specs/multiple_test.js'
   ],
   exclude: [],
 
   capabilities: [{
     name: 'multiple_appium_test',
     build: 'webdriver-browserstack',
-    device: 'iPhone 7',
+    device: 'iPhone 11 Pro',
```
Comment (darshilgada24) [inline ios/run_multiple_test/multiple.conf.js:14]: specify os_version

File: ios/run_local_test/local.conf.js [hunk 4]
```diff
@@ -6,20 +6,20 @@ exports.config = {
...
-    './tests/specs/local_test.js'
+    './run_local_test/specs/local_test.js'
   ],
   exclude: [],
 
   capabilities: [{
     name: 'local_appium_test',
     build: 'webdriver-browserstack',
-    device: 'iPhone 7',
+    device: 'iPhone 11 Pro',
```
Comment (darshilgada24) [inline ios/run_local_test/local.conf.js:16]: specify os_version

File: ios/run_first_test/specs/first_test.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    var textOutput = await $(`~Text Output`);
+    await textOutput.waitForDisplayed({ timeout: 30000 });
+    var value = await textOutput.getText();
+
+    if (value === "hello@browserstack.com")
+      assert(true)
+    else
+      assert(false)
+  });
+});
```
Comment (darshilgada24) [inline ios/run_first_test/specs/first_test.js:23]: add new line

File: ios/run_first_test/first.conf.js [hunk 6]
```diff
@@ -4,19 +4,19 @@ exports.config = {
...
     name: 'single_appium_test',
     build: 'webdriver-browserstack',
-    device: 'iPhone 7',
+    device: 'iPhone 11 Pro',
     app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>',
     'browserstack.debug': true
   }],
 
-  logLevel: 'verbose',
+  logLevel: 'info',
```
Comment (darshilgada24) [inline ios/run_first_test/first.conf.js:19]: Why the change from verbose to info?

File: ios/package.json [hunk 7]
```diff
@@ -5,10 +5,10 @@
   "description": "Selenium examples for WebdriverIO and BrowserStack App Automate",
   "scripts": {
     "test": "npm run single && npm run local && npm run parallel",
```
Comment (darshilgada24) [inline ios/package.json:7]: shouldn't this be `npm run first`

File: android/run_parallel_test/parallel.conf.js [hunk 8]
```diff
@@ -13,21 +13,19 @@ exports.config = {
     name: 'parallel_appium_test',
     build: 'webdriver-browserstack',
     browserName: 'android',
-    app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>',
+    app: process.env.BROWSERSTACK_APP_ID || 'bs://43e2a3ac988980aefbb17270c41df964c39a39c0',
```
Comment (darshilgada24) [inline android/run_parallel_test/parallel.conf.js:16]: change `bs://43e2a3ac988980aefbb17270c41df964c39a39c01`to 'bs://<hashed app-id>'

File: android/run_parallel_test/parallel.conf.js [hunk 9]
```diff
@@ -1,10 +1,10 @@
 exports.config = {
   user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
-  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACC_KEY',
+  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_USERNAME',
```
Comment (darshilgada24) [inline android/run_parallel_test/parallel.conf.js:3]: follow naming conventions from this PR https://github.com/kn-neeraj/python-appium-app-browserstack/blob/python3-readme-changes/android/examples/run-first-test/config.json

File: android/run_parallel_test/parallel.conf.js [hunk 10]
```diff
@@ -13,21 +13,19 @@ exports.config = {
     name: 'parallel_appium_test',
```
Comment (darshilgada24) [inline android/run_parallel_test/parallel.conf.js:13]: add project and build as well

File: android/run_multiple_test/multiple.conf.js [hunk 11]
```diff
@@ -4,20 +4,21 @@ exports.config = {
 
   updateJob: false,
   specs: [
-    './tests/specs/multiple_test.js'
+    './run_multiple_test/specs/multiple_test.js'
   ],
   exclude: [],
 
   capabilities: [{
     name: 'multiple_appium_test',
```
Comment (darshilgada24) [inline android/run_multiple_test/multiple.conf.js:12]: follow naming conventions and all capabilities required from https://github.com/kn-neeraj/python-appium-app-browserstack/blob/python3-readme-changes/android/examples/run-first-test/config.json

File: android/run_first_test/first.conf.js [hunk 12]
```diff
@@ -4,20 +4,21 @@ exports.config = {
 
   updateJob: false,
   specs: [
-    './tests/specs/single_test.js'
+    './run_first_test/specs/first_test.js'
   ],
   exclude: [],
 
   capabilities: [{
```
Comment (darshilgada24) [inline android/run_first_test/first.conf.js:11]: Follow naming conventions from https://github.com/kn-neeraj/python-appium-app-browserstack/blob/python3-readme-changes/android/examples/run-first-test/config.json

File: android/package.json [hunk 13]
```diff
@@ -5,10 +5,10 @@
   "description": "Selenium examples for WebdriverIO and BrowserStack App Automate",
   "scripts": {
     "test": "npm run single && npm run local && npm run parallel",
```
Comment (darshilgada24) [inline android/package.json:7]: shouldn't this be `npm run first`

File: README.md [hunk 14]
```diff
@@ -1,11 +1,60 @@
...
+cd ios
+npm i
+```
+
+## Getting Started
+
+Getting Started with Appium tests using WebdriverIO on BrowserStack couldn't be easier!
+
+### Run first test:
+  - Switch to `run_first_test` directory under [Android examples](./android) or [iOS examples](./ios)
```
Comment (darshilgada24) [inline README.md:36]: change this, since now it has changed to run-first-test. There will be other places as well, where you have to make these changes

File: android/run-first-test/first.conf.js [hunk 15]
```diff
@@ -1,23 +1,24 @@
 exports.config = {
   user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
```
Comment (darshilgada24) [inline android/run-first-test/first.conf.js:2]: change user to username and key to access_key. For reference, https://github.com/kn-neeraj/python-appium-app-browserstack/blob/python3-readme-changes/android/examples/run-first-test/config.json

File: android/run-first-test/first.conf.js [hunk 16]
```diff
@@ -1,23 +1,24 @@
...
-    './tests/specs/single_test.js'
+    './run-first-test/specs/first_test.js'
   ],
   exclude: [],
 
   capabilities: [{
     name: 'single_appium_test',
     build: 'webdriver-browserstack',
-    device: 'Samsung Galaxy S7',
+    device: 'Google Pixel 3',
```
Comment (darshilgada24) [inline android/run-first-test/first.conf.js:14]: project not added

File: android/run-local-test/local.conf.js [hunk 17]
```diff
@@ -2,25 +2,26 @@ var browserstack = require('browserstack-local');
 
 exports.config = {
   user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
```
Comment (darshilgada24) [inline android/run-local-test/local.conf.js:4]: Same as above comments

File: android/run-multiple-test/multiple.conf.js [hunk 18]
```diff
@@ -1,23 +1,24 @@
 exports.config = {
   user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
```
Comment (darshilgada24) [inline android/run-multiple-test/multiple.conf.js:2]: same as above comments

File: android/run-parallel-test/parallel.conf.js [hunk 19]
```diff
@@ -1,10 +1,10 @@
 exports.config = {
   user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
-  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACC_KEY',
+  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACCESS_KEY',
```
Comment (darshilgada24) [inline android/run-parallel-test/parallel.conf.js:3]: Same as above comments and for capabilities as well

File: ios/package-lock.json [hunk 20]
```diff
@@ -0,0 +1,2783 @@
+{
```
Comment (darshilgada24) [inline ios/package-lock.json:1]: any reason for committing package-lock.json

File: ios/run-first-test/first.conf.js [hunk 21]
```diff
@@ -1,22 +1,23 @@
 exports.config = {
   user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
-  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACC_KEY',
+  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACCESS_KEY',
```
Comment (darshilgada24) [inline ios/run-first-test/first.conf.js:3]: same comments as above for username and capabilities

File: ios/run_first_test/first.conf.js (hunk 6 above)
Comment (nurdtechie98) [reply ios/run_first_test/first.conf.js]: Yes, wdio v5 doesn't have loglevel verbose anymore it have to specify the level of verbose now
Options: trace | debug | info | warn | error | silent

File: android/run-first-test/first.conf.js (hunk 15 above)
Comment (nurdtechie98) [reply android/run-first-test/first.conf.js:2]: this kind of specific to webdriver it has to be user and key
https://webdriver.io/docs/options.html#user

**Review by darshilgada24 — APPROVED** [review]
